doc/manual/utils.nix

```diff
@@ -38,4 +38,46 @@ rec {
 
   filterAttrs = pred: set:
     listToAttrs (concatMap (name: let v = set.${name}; in if pred name v then [(nameValuePair name v)] else []) (attrNames set));
+
+  showSetting = useSpans: name: { description, documentDefault, defaultValue, aliases, ... }:
+    let
+      result = squash ''
+          - ${if useSpans
+              then ''<span id="conf-${name}">[`${name}`](#conf-${name})</span>''
```

Comment:
the #conf-foo anchors seem broken to me in $out/share/doc/nix/manual/command-ref/new-cli/nix3-help-stores.html

Reply:
Thanks, I've removed the anchors for now.